run_capoyo.sh

```
@@ -0,0 +1,15 @@
+#!/bin/bash
+#SBATCH --job-name=capoyo-full
+#SBATCH --nodes=1
+#SBATCH --gpus=4
+#SBATCH --mem-per-gpu=128G
+#SBATCH --cpus-per-gpu=16
+#SBATCH --output=logs/capoyo-full-%j.out
+#SBATCH --error=logs/capoyo-full-%j.err
+source /vast/projects/dyer1/lab/user/ian/envs/torch_brain_env/bin/activate
```

Shellcheck (0.11.0)

[info] Not following: /vast/projects/dyer1/lab/user/ian/envs/torch_brain_env/bin/activate was not specified as input (see shellcheck -x).

(SC1091)